Packs/Sysdig/Integrations/Sysdig/Sysdig.py

```diff
@@ -0,0 +1,337 @@
+from datetime import datetime
+
+import demistomock as demisto  # noqa: F401
+import requests
+from CommonServerPython import *  # noqa: F401
+from sdcclient import SdMonitorClient, SdSecureClient
+
+URL = demisto.params()['url']
+SECUREKEY = demisto.params()['securekey']
+MONITORKEY = demisto.params()['monitorkey']
+USE_SSL = not demisto.params().get('insecure', False)
```

Comment:
I suggest you should work with a client rather than saving the credentials into global params and create a client on each command execution to make your code more readable, you can use [this](https://github.com/demisto/content/blob/master/Packs/AzureKubernetesServices/Integrations/AzureKubernetesServices/AzureKubernetesServices.py) integration as a reference.